### PR TITLE
Add basic integration tests for button controls and text description

### PR DESCRIPTION
**Pull request by cathyxz**
For a single item lightbox, verify that: 
1. The description shows
2. The close button shows, and the remaining 3 buttons do not show.

### CODE REVIEW

File: extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js [hunk 1]
```diff
@@ -42,20 +42,9 @@ describe.configure().ifNewChrome().run('amp-lightbox-gallery', function() {
       win.AMP_MODE.localDev = true;
     });
 
-    it('should open and close correctly', () => {
+    it('should open and close correctly', done => {
```
Comment (cvializ) [inline extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js]: Returning the `openLightbox` Promise chains should be enough to tell Mocha that the test finished. When you use Promises, the `done` callback is almost always unnecessary

File: extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js [hunk 2]
```diff
@@ -74,11 +63,85 @@ describe.configure().ifNewChrome().run('amp-lightbox-gallery', function() {
...
+    const openerImage = ampImage.querySelector('img[amp-img-id="img0"]');
+    const openedPromise = waitForLightboxOpen(lightbox);
+    openerImage.click();
+    return openedPromise;
+  });
+}
+
+function getButton(document, className) {
+  const buttonQuery = document.getElementsByClassName(className);
+  expect(buttonQuery.length).to.equal(1);
```
Comment (cvializ) [inline extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js]: nit: `expect`s in getter helpers seems odd to me. It'd be OK to repeat the `expect` in each test

File: extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js (hunk 1 above)
Comment (cathyxz) [reply extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js]: I thought I saw some infra best practice notice about preferring `done` over returning Promises from @rsimha in some PR review that I once read, but I can't find it anymore. @rsimha do you have an opinion on return promise vs. calling `done()`?

Comment (rsimha) [reply extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js]: You could do either. If you're calling a function that returns a promise, simply return it from your test. If you need to do stuff after that, do your own promise synchronization and finally call done when the test can safely return.

Comment (rsimha) [reply extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js]: See https://mochajs.org/#asynchronous-code

**Pull request merged**